Velbus: Add information about the VMB7IN module (#29828)
home-assistant/home-assistant.github.io: source/_integrations/velbus.markdown

| `interface`            | no       | The port used to connect to the bus (the same one used during configuration). |
| `address`              | no       | The module address in decimal format, which is displayed on the device list on the integration page, if provided the service will only clear the cache for this model, without an address, the full velbuscache will be cleared. |

## VMB7IN and the Energy dashboard

> The VMB7IN does not repport what the counter is counting, because of this we need to let Home Assistant know what device class each counter is, in case the counter is counting an energy device everything will work out of the box.
> But in case the VMB7IN channel is a water or gas counter you will need to make some modifications in your configuration.yaml.

Reviewer: ```suggestion
In some cases, the VMB7IN sensor does not report what the counter is counting. If the counter is related to an energy device, everything will work out of the box.
But if the VMB7IN sensor is a water or gas counter, you will need to specify this in your configuration.yaml file.
```

```yaml
homeassistant:
  customize:

>     sensor.eau_counter:

Reviewer: is this called `eau_counter` both for water and gas?

Author: no this name can be whatever they configurd during the setup of there velbus system.

      device_class: water
```

The device_class attribute can have 2 values:

> - gas: in this case the counter is a gas meter
> - water: in this the counter is used to measure a water meter

Reviewer: ```suggestion
- gas: if the counter represents a gas meter
- water: if the counter represents a water meter
```
